src/mixxx.cpp

```diff
@@ -1079,6 +1105,15 @@ void MixxxMainWindow::slotNoMicrophoneInputConfigured() {
     m_pPrefDlg->showSoundHardwarePage();
 }
 
+void MixxxMainWindow::slotChangedPlayingDeck(int deck) {
+    UserSettingsPointer pConfig = m_pSettingsManager->settings();
+    int inhibit = pConfig->getValue<int>(ConfigKey("[Config]","InhibitScreensaver"));
```

Comment:
This can be a direct call, because the DlgPrefControls holds a pointer to MixxxMainWindow.